.github/workflows/build_and_test.yml

```diff
@@ -1644,26 +1634,12 @@ jobs:
         . setenv.sh
         cd tests/cmake_integration
         perl run_ci_tests.pl --skip-cxx11
-    - name: create OpenDDS tar.xz artifact
```

Comment:
Why was this part removed?

Reply:
Because we aren't using that build for testing so artifacting doesn't help with anything.

Reply:
Let's keep that kind of change in its own PR